config/mainCfg_ETau_UL17.cfg

```diff
@@ -0,0 +1,67 @@
+[general]
+lumi = 41529 # pb^-1 full 2017
+
+data = EGamma_Run2017B, EGamma_Run2017C, EGamma_Run2017D, EGamma_Run2017E, EGamma_Run2017F
+
```

Comment:
MET is missing here.

Reply:
Nicely spotted, thanks. I will use the names stored in ```config/sampleCfg_UL17.cfg```